test-data/Makefile

```diff
@@ -181,3 +181,19 @@ test-multiple-inputs:
      diff lognormalized-counts.txt ../../test-data/lognormalized-counts.txt && \
      diff barcode-counts.txt ../../test-data/barcode-counts.txt && \
      diff correlation.txt ../../test-data/correlation.txt)
+
+test-demultiplexed: wd = $(test-output-dir)/demultiplexed
+test-demultiplexed:
```

Comment:
The length limit for a unix command line is 4096 chars, which is starting to look achievable with this new demuxed input mode.  I guess that is all the more reason to switch to a JSON config file--was that just for pq-launcher or for pq as well?

Reply:
I wasn't planning to implement it in PoolQ in this PR. I think that is how PoolQ4 will work. In our case, the poolq launcher constructs the config object directly from _its_ command line, but I'm planning to change the poolq launcher command line to take only 2 parameters: `--db` and `--job-id`, which it will use to read everything it needs from the database, obviating the need for command lines that might exceed the limit.